src/sys/inotify.rs

```diff
@@ -0,0 +1,120 @@
+use libc::{
+    c_int,
+    uint32_t
+};
+use std::os::unix::io::RawFd;
+use {Error, Result};
+use NixPath;
+use errno::Errno;
+
+pub mod ffi {
```

Comment:
Nix no longer uses FFI directly.  C symbols should be placed in the libc crate instead.  Then Nix consumes libc.